app/ui/legacy/src/main/java/com/fsck/k9/ui/compose/QuotedMessagePresenter.java

```diff
@@ -154,17 +154,33 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     public void onRestoreInstanceState(Bundle savedInstanceState) {
-        quotedHtmlContent = (InsertableHtmlContent) savedInstanceState.getSerializable(STATE_KEY_HTML_QUOTE);
+        if (savedInstanceState != null) {
```

Comment:
Looking at the calls leading to this, shows indeed `savedInstanceState` can't be null. But `MessageCompose` and `QuotedMessagePresenter` don't declare `@NonNull` causing the warning. Will update them both.

Reply:
I don't know how the Java part was developed. There is usually either the assumption everything is non null by default and you need to state explicitly `@Nullable` or the oposite and you need to declare `@NonNull`. If we could make that explicit, it would simplify our `Java` to `Koltin` conversion efforts easier.

Reply:
To keep track of this and to continue discussion, I created the ticket #762